app/views/forms/_odk_group.xml.erb

```diff
@@ -4,7 +4,11 @@
   <% grid_mode = grid_mode?(children) %>
 
   <group appearance="field-list">
-    <label><%= group.group_name %></label>
+    <label ref="jr:itext('name<%= group.id %>:label')"/>
```

Comment:
In testing it looks like if the label is here, it is not bold. If you move it inside the input tag below, it will become bold and look better.

Then, however, this really becomes a "group header" device, not group hint. So I would change all the names to e.g. grp-header instead of hint. Make sense?